### DISCUSSION TITLE
Input Group

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#296] [Component suggestions]
<img width="480" alt="visuele weergave van de input group component" src="https://github.com/user-attachments/assets/d1d74997-dc13-45d0-a94e-bd277ddcd804" />

We willen graag meer leren over de Input Group component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, GitHub of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1733745604200079)

---

## Naam
Input Group

## Doel
Formulierveld waarmee je een waarde kunt bewerken in een groep van meerdere invoervelden.

## Ook bekend als
Text Input Group, Control Group.

## Gerelateerde componenten
[Date Input Group](https://github.com/orgs/nl-design-system/discussions/271), [Form Field Partial](https://github.com/orgs/nl-design-system/discussions/314), [Form Field](https://github.com/orgs/nl-design-system/discussions/311), [Text Input](https://github.com/orgs/nl-design-system/discussions/225)

## Onderzoek

**Comment by jeffreylauwers** [comment]
## DUO

Niet als los component gevonden. Maar als als onderdeel van de Form component.

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/form)

---

### Screenshots / Varianten

**Documentatie website**

Default
![Scherm­afbeelding 2024-12-05 om 11 10 43](https://github.com/user-attachments/assets/0a123779-e378-4a0a-9a6c-26ac2af80c76)

Filled + Focus
![Scherm­afbeelding 2024-12-05 om 11 11 40](https://github.com/user-attachments/assets/bd8075b2-c289-41d0-b86c-b29370648588)

---

### Notities

- Binnen een `fieldset` met `legend`.
- CSS class van groep invulvelden is `input__control-group`.
- CSS class van enkel invulveld binnen deze groep is `input__control-container`.
- Deze aanpak wordt ook toegepast bij de 'Date input' component zoals te zien in de [Date Input Group Github Discussion](https://github.com/orgs/nl-design-system/discussions/271#discussioncomment-10981095).

**Comment by jeffreylauwers** [comment]
## Open Formulieren

[Storybook](https://open-formulieren.github.io/open-forms-sdk/?path=/docs/pure-react-components-forms-low-level-input-group--docs)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-12-05 om 11 44 36](https://github.com/user-attachments/assets/0c16d3c0-07b8-4f75-944c-f93e2832f5ea)

Filled + Focus
![Scherm­afbeelding 2024-12-05 om 11 48 16](https://github.com/user-attachments/assets/52fe7bae-a836-4b7c-9f43-1c94b16408e3)

---

### Notities

- Binnen een `fieldset` met `legend` (met daarbinnen weer een `label`?).
- CSS class van groep invulvelden is `openforms-input-group`.
- CSS class van enkel invulveld binnen deze groep is `openforms-input-group__item`.
- Deze aanpak wordt ook toegepast bij de 'DateField - input group' component zoals te zien in de [Date Input Group Github Discussion](https://github.com/orgs/nl-design-system/discussions/271#discussioncomment-11461854).

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Input Group op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=input+group+178)
